### PR TITLE
e2e: clean up to test tables, endpoint-health test

### PR DESCRIPTION
**Pull request by gyuho**
/cc @heyitsanthony 

Thanks.

### CODE REVIEW

File: e2e/etcdctlv3_test.go [hunk 1]
```diff
@@ -242,47 +221,74 @@ func getTest(cx ctlCtx) {
...
-		keyToDel   = "key"
-		deletedNum = 3
-	)
-	if !cx.prefix {
-		keyToDel = "key1"
-		deletedNum = 1
+	tests := []struct {
+		puts       []kv
+		args       []string
+		deletedNum int
```
Comment (heyitsanthony) [inline e2e/etcdctlv3_test.go]: newline between inputs and expected outputs?

File: e2e/etcdctlv3_test.go [hunk 2]
```diff
@@ -242,47 +221,74 @@ func getTest(cx ctlCtx) {
...
+	}{
+		{
```
Comment (heyitsanthony) [inline e2e/etcdctlv3_test.go]: 0 deleted test too?

File: e2e/etcdctlv3_test.go [hunk 3]
```diff
@@ -242,47 +221,74 @@ func getTest(cx ctlCtx) {
...
-	}()
-
-	keyToWatch := "key"
-	if !cx.prefix {
-		keyToWatch = "key1"
-		kvs = kvs[:1]
+	tests := []struct {
+		puts []kv
+		args []string
+		wkv  []kv
```
Comment (heyitsanthony) [inline e2e/etcdctlv3_test.go]: newline between args and wkv?

File: e2e/etcdctlv3_test.go [hunk 4]
```diff
@@ -295,6 +301,18 @@ func versionTest(cx ctlCtx) {
...
 }
 
+func epHealthTest(cx ctlCtx) {
+	defer close(cx.errc)
+
+	clusterSize := 1
+	if cx.quorum {
+		clusterSize = 3
+	}
+	if err := ctlV3EpHealth(cx, clusterSize); err != nil {
```
Comment (heyitsanthony) [inline e2e/etcdctlv3_test.go]: `cx.epc.cfg.clusterSize`?

Comment (gyuho) [inline e2e/etcdctlv3_test.go]: right. Will change. Thanks.

**Comment by gyuho** [comment]
@heyitsanthony PTAL. Thanks!

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
Thanks!

**Pull request merged**